src/Webapi/Webapi__Dom.re

```diff
@@ -64,10 +67,14 @@ module ValidityState = Webapi__Dom__ValidityState;
 module WebGlContextEvent = Webapi__Dom__WebGlContextEvent;
 module WheelEvent = Webapi__Dom__WheelEvent;
 module Window = Webapi__Dom__Window;
+module Worker = Webapi__Dom__Worker;
+module WorkerGlobalScope = Webapi__Dom__WorkerGlobalScope;
+module DedicatedWorkerGlobalScope = Webapi__Dom__DedicatedWorkerGlobalScope;
 
 include Webapi__Dom__Types;
 
 external window: Dom.window = "window";
+external dedicatedWorkerGlobalScope: Dom.dedicatedWorkerGlobalScope = "self";
```

Comment:
@davesnx Is it better to have a `globalThis` here? Like

```reason
external globalThis: 'a = "globalThis"
```

Reply:
Yes, we should add globalThis